paint.js

```diff
@@ -101,8 +107,21 @@ function PointerHandler(event) {
         };
         var eventType = event.type == "pointerdown" ? "touchstart" :
             event.type == "pointerup" ? "touchend" : "touchmove";
-        drawTouch(fakeTouch, eventType);
         event.preventDefault();
+        drawTouch(fakeTouch, eventType, false);
```

Comment:
Although this works just fine for this drawing app but that was not how we encouraged people to use it in the spec. So basically right now we are drawing two circles at the same point since the last coalesced event will have the same coordinates as the event itself. If people look at this example on how to use coalesced events they might get confused a little.